crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -9,11 +9,13 @@ use bevy_render::{
 };
 use std::collections::HashMap;
 
+/// An atlas containing multiple textures in one asset (like a spritesheet or a tilemap)
```

Comment:
Lets just cut `in one asset` from the end. Theres always a possibility that this will get used in a non-asset context.